ActionLogger/src/pl/jtat/logger/logwriter/LocalDestinationLogWriter.java

```diff
@@ -0,0 +1,34 @@
+package pl.jtat.logger.logwriter;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class LocalDestinationLogWriter extends LogWriter {
+
+    private String localLogFilePath;
+
+    public LocalDestinationLogWriter(String localLogFilePath) {
+        this.localLogFilePath = localLogFilePath;
+    }
+
+    @Override
+    public void saveLog(String formattedEventData) {
+        System.out.println(formattedEventData);
+
+        File logFile = new File(localLogFilePath);
```

Comment:
Będziemy chcieli docelowo zapisywać logi do bazy danych, żeby frontend mógł uzyskać do nich dostęp.
Tak więc implementacja LogWritera powinna przyjmować do zapisu EventData, a nie wygenerowany string, bo później ciężko to będzie przetworzyć na froncie.

Reply:
Parametr zostanie zmieniony lub zostanie dopisana funkcja (na chwilą obecną i tak nie łączy się z takową bazą więc warto żeby coś zostało takiego do testowania/wyrzucania na konsolę dla kolejnej możliwej implementacji LogWritera).

Reply:
Nie rozumiem w czym ci to pomaga w testowaniu. Wystarczy zamiast w CinemaActionLogger robić parseDataToString() to robić to tutaj

Reply:
Widzę, że nie rozumiesz :) w kolejnej iteracji zostanie dodana implementacja LogWritera, która będzie się nazywać DatabaseLogWriter z, tak jak napisałam, zmienionym typem parametru lub dodatkową funkcją w klasie implementującej.

Reply:
To niech interfejs już w tym PR posiada metodę z odpowiednim typem parametru.
Jak już robisz interfejs to tak, żeby istniejących w nim metod już nie zmieniać